### ISSUE TITLE
getting a ukvm-bin with the gdb module

### ISSUE THREAD

**Issue by djwillia** [Solo5/solo5#156]
At the moment, using the "normal" build for Solo5 or Mirage, we'll end up with a `ukvm-bin` that doesn't have the gdb module in it (which makes sense).  However, it's a bit cumbersome to attach gdb if you do want to debug it via gdb.  I've been editing the generated Makefile.ukvm (https://github.com/Solo5/solo5/blob/master/ukvm/ukvm-configure#L38) by hand if I want to attach gdb through that `ukvm-bin`, e.g.:
```
UKVM_MODULE_OBJS=$(addsuffix .o,$(addprefix _build-ukvm/ukvm-,)) _build-ukvm/ukvm-gdb.o
UKVM_MODULE_FLAGS=$(addprefix -DUKVM_MODULE_,$(shell echo | tr '[:lower:]' '[:upper:]')) -DUKVM_MODULE_GDB
```
Perhaps it makes more sense to have a `ukvm-bin-gdb` target or similar be generated in the Makefile that already specifies the gdb module?  What do others think?

**Comment by ricarkol** [comment]
What about having the Makefile always generate both versions of ukvm: `ukvm-bin` and `ukvm-bin-gdb`. Otherwise, telling `mirage` to generate the gdb versions might involve changes to the `mirage` tool.

**Comment by mato** [comment]
Fixed in mirage/mirage#847.
